addons/mail/models/res_partner.py

```diff
@@ -21,10 +20,13 @@ class Partner(models.Model):
 
     email = fields.Char(tracking=1)
     phone = fields.Char(tracking=2)
+    portal_access = fields.Boolean('Portal Access', compute="_compute_portal_access", tracking=True, store=True)
```

Comment:
I would not add a stored computed tracked field just to log when the "Portal Access" change... It will add a SQL column in the table and this model is already big...

Maybe you can log manually in the create / write methods of `res.users` ?
What do you think ?

(I open to other suggestion if you find a cool idea without a stored field)

Reply:
Hello @std

it was easy to manage this with a stored compute field.
if we try to manage this with create/write then it will create a separate lognote without an entry in the model used for the track. So we need to manage everything on our own for that (not sure but might send mails also sometimes if misconfigured).
And also we have to manage commands in the values of it.
either the command is 4,5,6 and then compare it with the id of the portal user.
And it will make the create/write much for complex and prone to break.
that's why we have added this stored compute field. 

Thanks

CC: @LNAodoo @awa-odoo

Reply:
@nla-odoo To know if the value change in the write, you do not need to parse the "magic Odoo numbers", you can just call "has_group" before the "super()" (in the write method) and then recall "has_group" after the "super()".

**Or** you can keep your computed method, but remove the `store=True` and force the tracking of the field in the write method of `res.users` by calling

```python
self.partner_id._prepare_tracking(['portal_access'])
```

In the end I do not like adding a stored field on `res.partner` just to log to change on one group of the related user... 

Maybe @tde-banana-odoo has an other good trick to be able to log non-stored field ?

Reply:
Or rather just log a message in the chatter : "Bob become portal" or something like, it would be even better IMO